x-pack/legacy/plugins/siem/index.ts

```diff
@@ -130,6 +130,17 @@ export const siem = (kibana: any) => {
         },
       },
       mappings: savedObjectMappings,
+      // TODO: Remove once while Saved Object Mappings API is programmed for the NP See: https://github.com/elastic/kibana/issues/50309
+      savedObjectSchemas: {
```

Comment:
@XavierM I'm wondering if I should not commit this file. I was thinking we could listen for my kibana.yml var but i dont think we get access to that until the init

Reply:
another question... I thought I could have 2 different object types to the same index pattern, is that not so? When I run my server it creates 2 index patterns, `.case-testing-ground_1` and `.case-testing-ground_2`. I wasn't expecting that

Reply:
Also...... should `isNamespaceAgnostic` be true? I read that code but did not understand the effects??

Reply:
I think that we want it to be false because after talking to @kobelb to make sure that I understood correctly. Anyway, if `isNamespaceAgnostic: false` the case will only be viewable in the space where it has been created.